sdk/docs/iot/README.md

```diff
@@ -54,9 +54,18 @@ For more detailed, step-by-step guides to setup and run the IoT Hub certificate
 - Espressif ESP32: [How to Setup and Run Azure SDK for Embedded C IoT Hub Client on ESP32](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/aziot_esp32/readme.md)
 
 - Azure Device Update with Espressif ESP32 on Arduino IDE: [How to Setup and Run Azure SDK for Embedded C ADU on Espressif ESP32](https://github.com/Azure/azure-sdk-for-c-arduino/blob/main/examples/Azure_IoT_Adu_ESP32/readme.md)
-
 - VxWorks: [How to Setup and Run Azure SDK for Embedded C IoT Hub Client and Provisioning Client Samples on VxWorks](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_vxworks.md)
```

Comment:
move VxWorks under here as well?

Reply:
this lives in our repo though right? These new guidelines are for links to external ones?

Reply:
While the guidelines will be effective to all new samples, the VXWorks is also a community contribution so moving it makes a clear distinction between the ones we own and the remaining ones.